ros_gz_bridge/ros_gz_bridge/actions/ros_gz_bridge.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         :param: log_level Log level.
         """
         super().__init__(**kwargs)
-        self.__name = name
+        self.__bridge_name = bridge_name
```

Comment:
Nit: Could you update the documentation a few lines above?

Reply:
done